<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainformClass</class>
 <widget class="QMainWindow" name="mainformClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainform</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="mainTab">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="initiation">
       <attribute name="title">
        <string>初始化</string>
       </attribute>
       <widget class="glViewer" name="init_viewer" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>581</width>
          <height>557</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
       <widget class="QToolBox" name="init_toolbox">
        <property name="geometry">
         <rect>
          <x>598</x>
          <y>11</y>
          <width>289</width>
          <height>555</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>211</width>
           <height>174</height>
          </rect>
         </property>
         <attribute name="label">
          <string>扫描参数</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="delaylabel">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>启动延时</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="delay_slider">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximum">
                   <number>15</number>
                  </property>
                  <property name="pageStep">
                   <number>5</number>
                  </property>
                  <property name="value">
                   <number>5</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="delay_value">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>5</string>
                  </property>
                  <property name="margin">
                   <number>5</number>
                  </property>
                  <property name="indent">
                   <number>10</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>s</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="rangelabel">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>扫描范围</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="range_slider">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximum">
                   <number>2000</number>
                  </property>
                  <property name="singleStep">
                   <number>50</number>
                  </property>
                  <property name="pageStep">
                   <number>500</number>
                  </property>
                  <property name="value">
                   <number>1500</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="range_value">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>1500</string>
                  </property>
                  <property name="margin">
                   <number>5</number>
                  </property>
                  <property name="indent">
                   <number>10</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>mm</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>3</number>
              </property>
              <item>
               <widget class="QRadioButton" name="full_rb">
                <property name="text">
                 <string>全身</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="half_rb">
                <property name="text">
                 <string>半身</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="hea_rb">
                <property name="text">
                 <string>头像</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>14</width>
                <height>268</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="defaultBtn">
                <property name="text">
                 <string>恢复默认</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>287</width>
           <height>475</height>
          </rect>
         </property>
         <attribute name="label">
          <string>Kinfu参数</string>
         </attribute>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="scan">
       <attribute name="title">
        <string>扫描</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="0">
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,1">
            <item>
             <widget class="glViewer" name="fusionViewer" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>2</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="glViewer" name="depthViewer" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>2</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <widget class="glViewer" name="RGBViewer" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>2</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <widget class="glViewer" name="deviceViewer" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>2</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>控制台输出</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QTextEdit" name="textEdit"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="recordBtn">
              <property name="text">
               <string>开始</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stopBtn">
              <property name="text">
               <string>停止</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteBtn">
              <property name="text">
               <string>清除</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="showCloudBtn">
              <property name="text">
               <string>显示点云</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="reconstruction">
       <attribute name="title">
        <string>三维重建</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,1">
          <item>
           <widget class="glViewer" name="init_viewer_6" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="1">
              <widget class="QToolBox" name="toolBox">
               <property name="font">
                <font>
                 <family>微软雅黑</family>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="page_3">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>98</width>
                  <height>28</height>
                 </rect>
                </property>
                <attribute name="label">
                 <string>操作</string>
                </attribute>
                <widget class="QPushButton" name="pushButton">
                 <property name="geometry">
                  <rect>
                   <x>50</x>
                   <y>40</y>
                   <width>171</width>
                   <height>71</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>重建</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="pushButton_2">
                 <property name="geometry">
                  <rect>
                   <x>50</x>
                   <y>150</y>
                   <width>171</width>
                   <height>71</height>
                  </rect>
                 </property>
                 <property name="text">
                  <string>复位</string>
                 </property>
                </widget>
               </widget>
               <widget class="QWidget" name="page_4">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>98</width>
                  <height>28</height>
                 </rect>
                </property>
                <attribute name="label">
                 <string>重建参数</string>
                </attribute>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="process">
       <attribute name="title">
        <string>后处理</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,1">
          <item>
           <widget class="glViewer" name="init_viewer_2" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,1">
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>模型操作</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_8">
               <item row="0" column="0">
                <widget class="QToolBox" name="toolBox_2">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                   <weight>50</weight>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="page_5">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>98</width>
                    <height>28</height>
                   </rect>
                  </property>
                  <attribute name="label">
                   <string>模型简化</string>
                  </attribute>
                 </widget>
                 <widget class="QWidget" name="page_6">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>98</width>
                    <height>28</height>
                   </rect>
                  </property>
                  <attribute name="label">
                   <string>生成实体模型</string>
                  </attribute>
                 </widget>
                 <widget class="QWidget" name="page_7">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>98</width>
                    <height>28</height>
                   </rect>
                  </property>
                  <attribute name="label">
                   <string>修补孔洞</string>
                  </attribute>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>输出</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>扫描</string>
    </property>
    <addaction name="ToolstartBtn"/>
    <addaction name="ToolstopBtn"/>
    <addaction name="TooldeleteBtn"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionSaveCloud"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="connectKinect"/>
   <addaction name="connectlifter"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ToolstartBtn"/>
   <addaction name="ToolstopBtn"/>
   <addaction name="TooldeleteBtn"/>
   <addaction name="separator"/>
   <addaction name="actionSaveCloud"/>
  </widget>
  <action name="connectKinect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mainform.qrc">
     <normaloff>:/images/Resources/images/260_64.png</normaloff>:/images/Resources/images/260_64.png</iconset>
   </property>
   <property name="text">
    <string>连接Kinect</string>
   </property>
  </action>
  <action name="connectlifter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mainform.qrc">
     <normaloff>:/images/Resources/images/2215_64.png</normaloff>:/images/Resources/images/2215_64.png</iconset>
   </property>
   <property name="text">
    <string>连接升降台</string>
   </property>
  </action>
  <action name="ToolstartBtn">
   <property name="icon">
    <iconset resource="mainform.qrc">
     <normaloff>:/images/Resources/images/start.png</normaloff>:/images/Resources/images/start.png</iconset>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="ToolstopBtn">
   <property name="icon">
    <iconset>
     <normalon>:/images/Resources/images/stop.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="TooldeleteBtn">
   <property name="icon">
    <iconset>
     <normalon>:/images/Resources/images/recycle.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </action>
  <action name="actionSaveCloud">
   <property name="text">
    <string>保存点云</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>glViewer</class>
   <extends>QWidget</extends>
   <header location="global">GL2Dviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="mainform.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>delay_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>delay_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>715</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>772</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>range_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>range_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>777</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
